archive-zip.c

```diff
@@ -352,7 +352,7 @@ static int write_zip_entry(struct archiver_args *args,
 			if (!buffer)
 				return error(_("cannot read %s"),
 					     oid_to_hex(oid));
-			crc = crc32(crc, buffer, size);
+			crc = xcrc32(crc, buffer, size);
```

Comment:
I think this makes sense, but the sentence "Zero length initialisations are not converted." is *very* cryptic.

Reply:
Any better suggestions. Zero length fits into both 32bit and 64bit longs ;-)

Reply:
I have *no* idea what you mean by that.

Reply:
(That sentence makes as much sense to me as "Correct horse battery staple".)

Reply:
maybe? Zero length initialisations work with both 32bit and 64bit `uInt len` values ;-) (only slightly less cryptic)

Otherwise it's a lot of noise changes for no benefit (there are quite a few initialisations..).